packages/dev/core/src/Engines/WebGPU/webgpuShaderProcessorsWGSL.ts

```diff
@@ -494,4 +508,11 @@ export class WebGPUShaderProcessorWGSL extends WebGPUShaderProcessor {
 
         return code;
     }
+
+    private _processStridedUniformArrays(code: string): string {
+        for (const uniformArrayName of this._stridedUniformArrays) {
+            code = code.replace(new RegExp(`${uniformArrayName}\\[(.*)\\]`, "g"), `${uniformArrayName}[$1].el`);

```

Comment:
I think syntax like `myarray [23]` is allowed in wgsl, so a `\s*` should be added in the regexp?

Something like: `${uniformArrayName}\\s*\\[(.*)\\]`

Reply:
yup